helm/alfresco-content-services-community/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: alfresco-content-services-community
-version: 2.2.0
+version: 3.0.0
```

Comment:
I think 3.0.0 should be reserved for 6.2.X. As we still have 6.1 here, it should really be 2.2.1 for now.

Reply:
Based on helm-versioning document -> https://github.com/Alfresco/acs-deployment/blob/master/docs/helm-versioning.md , 3.0.0 should be reserved for 6.2.X, totally agree :) .

Acs-packaging master branch is 6.2.0-SNAPSHOT -> https://github.com/Alfresco/acs-packaging/blob/master/pom.xml
After I have released acs-community packaging, I set master branch to 6.2.0-SNAPSHOT ->https://github.com/Alfresco/acs-community-packaging/blob/master/pom.xml
Just to have both in sync.

Next version of community can be 6.1.3-ga (Chart version of 2.3.0) either 6.2.0-ea(Chart version of 3.0.0)
We don't have hot-fixes on community (Chart version of 2.2.1 should be for 6.1.2.1, which is not applicable in this case), I`m fine to setting the chart version to 2.3.0 instead of 3.0.0.

Reply:
I see, we skipped a version. Then it is ok to keep in sync with enterprise.